lib/SILGen/SILGenFunction.h

```diff
@@ -2002,6 +1995,13 @@ class LLVM_LIBRARY_VISIBILITY SILGenFunction
   void initializeDistributedActorImplicitStorageInit(
       ConstructorDecl *ctor, ManagedValue selfArg);
 
+  /// Given a function representing a distributed actor factory, emits the
+  /// corresponding SIL function for it.
+  void emitDistributedActorFactory(FuncDecl *fd);
+
+  /// Generates a thunk from an actor function
+  void emitDistributedThunk(SILDeclRef thunk);
+
```

Comment:
moved them together under a ******** header we have for all distributed ones here